internal/middleware/middleware.go

```diff
@@ -0,0 +1,51 @@
+package middleware
+
+import (
+	"context"
+	"github.com/google/uuid"
+	"github.com/sirupsen/logrus"
+	"net/http"
+	"time"
+)
+
+var Entry *logrus.Entry
```

Comment:
Since the entry logger can be used across the whole app I'd say it should be at least in the main package. 
Or will be even better if you init entry logger and pass it as an argument to all application layers through DI.